<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1233</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_11" stretch="1,1,8">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,1,1,0,1,0,1,0,0,1">
        <item>
         <widget class="QLabel" name="transducer_label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;传感器&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;0.0um&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;范围：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_2">
          <item>
           <property name="text">
            <string>+-50μm</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>+-500μm</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>+-2000μm</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;主轴：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;0.0rpm&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;0°&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;MS：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1233</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>新建</string>
     </property>
     <addaction name="new_profile"/>
     <addaction name="measurement_task"/>
    </widget>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>打开</string>
     </property>
     <addaction name="open_configuration"/>
     <addaction name="open_measurement_job"/>
     <addaction name="action1_11"/>
    </widget>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>打印</string>
     </property>
     <addaction name="action1_12"/>
     <addaction name="action1_13"/>
    </widget>
    <widget class="QMenu" name="menu_8">
     <property name="title">
      <string>另存为</string>
     </property>
     <addaction name="save_as_configuration"/>
     <addaction name="action1_4"/>
     <addaction name="action1_15"/>
    </widget>
    <addaction name="menu_2"/>
    <addaction name="menu_3"/>
    <addaction name="menu_8"/>
    <addaction name="actionq"/>
    <addaction name="action1_3"/>
    <addaction name="menu_4"/>
    <addaction name="action1_5"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>测量</string>
    </property>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>校准</string>
     </property>
     <addaction name="action1_17"/>
     <addaction name="action1_18"/>
     <addaction name="action1_19"/>
    </widget>
    <widget class="QMenu" name="menu_7">
     <property name="title">
      <string>删除</string>
     </property>
     <addaction name="action1_20"/>
     <addaction name="action1_21"/>
     <addaction name="action1_22"/>
    </widget>
    <addaction name="start_measuring"/>
    <addaction name="menu_6"/>
    <addaction name="menu_7"/>
   </widget>
   <widget class="QMenu" name="menu_9">
    <property name="title">
     <string>访问权限</string>
    </property>
    <addaction name="actionq_2"/>
    <addaction name="actionq_3"/>
    <addaction name="actionq_4"/>
   </widget>
   <widget class="QMenu" name="menu_10">
    <property name="title">
     <string>配置</string>
    </property>
    <addaction name="config_action1"/>
    <addaction name="action1_23"/>
    <addaction name="action12"/>
    <addaction name="action12_2"/>
    <addaction name="action12_3"/>
    <addaction name="calibration_password"/>
    <addaction name="action12_5"/>
    <addaction name="action12_6"/>
    <addaction name="action12_7"/>
    <addaction name="action12_8"/>
    <addaction name="action1_24"/>
   </widget>
   <widget class="QMenu" name="menu_11">
    <property name="title">
     <string>窗口</string>
    </property>
    <addaction name="polar_diagram"/>
    <addaction name="spectrogram"/>
    <addaction name="action1_27"/>
    <addaction name="statistical_report"/>
    <addaction name="action1_29"/>
   </widget>
   <widget class="QMenu" name="menu_12">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action1_30"/>
   </widget>
   <widget class="QMenu" name="menu_13">
    <property name="title">
     <string>对象</string>
    </property>
    <addaction name="action1_31"/>
    <addaction name="action1_32"/>
    <addaction name="action1_33"/>
    <addaction name="action1_34"/>
    <addaction name="action1_35"/>
    <addaction name="action1_36"/>
    <addaction name="action1_37"/>
    <addaction name="action1_38"/>
    <addaction name="action1_39"/>
    <addaction name="action1_40"/>
    <addaction name="action1_41"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_5"/>
   <addaction name="menu_9"/>
   <addaction name="menu_10"/>
   <addaction name="menu_13"/>
   <addaction name="menu_11"/>
   <addaction name="menu_12"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionq">
   <property name="text">
    <string>最近使用的文件</string>
   </property>
  </action>
  <action name="action1_3">
   <property name="text">
    <string>导出数据</string>
   </property>
  </action>
  <action name="action1_5">
   <property name="text">
    <string>历史记录</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="new_profile">
   <property name="text">
    <string>配置</string>
   </property>
   <property name="shortcut">
    <string>Shift+F1</string>
   </property>
  </action>
  <action name="measurement_task">
   <property name="text">
    <string>测量作业</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="open_configuration">
   <property name="text">
    <string>配置</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="open_measurement_job">
   <property name="text">
    <string>测量作业</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="action1_11">
   <property name="text">
    <string>测量数据</string>
   </property>
  </action>
  <action name="action1_12">
   <property name="text">
    <string>窗口</string>
   </property>
  </action>
  <action name="action1_13">
   <property name="text">
    <string>报告</string>
   </property>
  </action>
  <action name="start_measuring">
   <property name="text">
    <string>开始测量</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="action1_17">
   <property name="text">
    <string>主控件</string>
   </property>
  </action>
  <action name="action1_18">
   <property name="text">
    <string>测量盒</string>
   </property>
  </action>
  <action name="action1_19">
   <property name="text">
    <string>编辑校准模板</string>
   </property>
  </action>
  <action name="action1_20">
   <property name="text">
    <string>上次测量</string>
   </property>
  </action>
  <action name="action1_21">
   <property name="text">
    <string>全部MS的统计</string>
   </property>
  </action>
  <action name="action1_22">
   <property name="text">
    <string>上次测量的统计</string>
   </property>
  </action>
  <action name="save_as_configuration">
   <property name="text">
    <string>配置</string>
   </property>
  </action>
  <action name="action1_4">
   <property name="text">
    <string>测量作业</string>
   </property>
  </action>
  <action name="action1_15">
   <property name="text">
    <string>测量数据</string>
   </property>
  </action>
  <action name="actionq_2">
   <property name="text">
    <string>标准访问权限</string>
   </property>
  </action>
  <action name="actionq_3">
   <property name="text">
    <string>完全访问权限</string>
   </property>
  </action>
  <action name="actionq_4">
   <property name="text">
    <string>校准访问权限</string>
   </property>
  </action>
  <action name="config_action1">
   <property name="text">
    <string>配置</string>
   </property>
  </action>
  <action name="action1_23">
   <property name="text">
    <string>报告</string>
   </property>
  </action>
  <action name="action12">
   <property name="text">
    <string>自动保存</string>
   </property>
  </action>
  <action name="action12_2">
   <property name="text">
    <string>插件管理器</string>
   </property>
  </action>
  <action name="action12_3">
   <property name="text">
    <string>密码</string>
   </property>
  </action>
  <action name="calibration_password">
   <property name="text">
    <string>校准</string>
   </property>
  </action>
  <action name="action12_5">
   <property name="text">
    <string>机器</string>
   </property>
  </action>
  <action name="action12_6">
   <property name="text">
    <string>字体</string>
   </property>
  </action>
  <action name="action12_7">
   <property name="text">
    <string>风格</string>
   </property>
  </action>
  <action name="action12_8">
   <property name="text">
    <string>语言</string>
   </property>
  </action>
  <action name="action1_24">
   <property name="text">
    <string>全屏</string>
   </property>
  </action>
  <action name="polar_diagram">
   <property name="text">
    <string>极图</string>
   </property>
  </action>
  <action name="spectrogram">
   <property name="text">
    <string>光谱</string>
   </property>
  </action>
  <action name="action1_27">
   <property name="text">
    <string>定制化报告</string>
   </property>
  </action>
  <action name="statistical_report">
   <property name="text">
    <string>统计</string>
   </property>
  </action>
  <action name="action1_29">
   <property name="text">
    <string>日志记录</string>
   </property>
  </action>
  <action name="action1_30">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action1_31">
   <property name="text">
    <string>极图</string>
   </property>
  </action>
  <action name="action1_32">
   <property name="text">
    <string>调和曲线图</string>
   </property>
  </action>
  <action name="action1_33">
   <property name="text">
    <string>线性曲线图</string>
   </property>
  </action>
  <action name="action1_34">
   <property name="text">
    <string>结果</string>
   </property>
  </action>
  <action name="action1_35">
   <property name="text">
    <string>测量信息</string>
   </property>
  </action>
  <action name="action1_36">
   <property name="text">
    <string>识别</string>
   </property>
  </action>
  <action name="action1_37">
   <property name="text">
    <string>文本</string>
   </property>
  </action>
  <action name="action1_38">
   <property name="text">
    <string>标志</string>
   </property>
  </action>
  <action name="action1_39">
   <property name="text">
    <string>复制对象</string>
   </property>
  </action>
  <action name="action1_40">
   <property name="text">
    <string>删除对象</string>
   </property>
  </action>
  <action name="action1_41">
   <property name="text">
    <string>插入对象</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
